web-client/src/presenter/actions/StartCase/updatePartyTypeActionUpdated.ts

```diff
@@ -0,0 +1,97 @@
+import { state } from '@web-client/presenter/app.cerebral';
+
+export const updatePartyTypeActionUpdated = ({
+  applicationContext,
+  props,
+  store,
+}: ActionProps<{
+  key: string;
+  value: string;
+}>) => {
+  const { PARTY_TYPES } = applicationContext.getConstants();
+
+  const resetFormFields = () => {
+    store.unset(state.form.corporateDisclosureFile);
+    store.unset(state.form.corporateDisclosureFileSize);
+    store.unset(state.form.businessType);
+    store.unset(state.form.otherType);
+    store.unset(state.form.isSpouseDeceased);
+    store.unset(state.form.estateType);
+    store.unset(state.form.minorIncompetentType);
+    store.unset(state.form.hasSpouseConsent);
+  };
+
+  const updatePartyType = (newPartyType: string) => {
+    store.set(state.form.partyType, newPartyType);
+  };
+
+  const handleFilingType = () => {
```

Comment:
Some of these handlers use destructuring syntax to access the props' value property, while others don't. It might be good to be consistent.